Source/WebCore/platform/graphics/TiledBacking.h

```diff
@@ -94,6 +103,11 @@ class TiledBacking : public CanMakeCheckedPtr<TiledBacking> {
 
     virtual PlatformLayerIdentifier layerIdentifier() const = 0;
 
+    virtual std::optional<TileConfigurationChangeIdentifier> activeConfigurationChange() const = 0;
+    virtual TileConfigurationChangeIdentifier makeTileConfigurationChange(CompletionHandler<void(TileGridIdentifier oldGrid, TileGridIdentifier newGrid, DidCancelTileConfigurationChange)>&&) = 0;
+    virtual void didPrepareContentForTile(TileIndex) = 0;
+    virtual void swapPrimaryGridWithPendingTileGrid(TileGridIdentifier oldGrid, TileGridIdentifier newGrid) = 0;
```

Comment:
The way I've structured the flow now, we need to know the old/new grids before `commitTileConfigurationChange` is called (since this would be the last step that actually does the grid swap).

So I'm renaming the method, to abstract away from the tile grid swap mechanics, but I'm keeping the signature `void(TileGridIdentifier, TileGridIdentifier)` for now.